### ISSUE TITLE
Target global position has an offset when using rootOverlay and inner navigator

### ISSUE THREAD

**Issue by guildem** [RafaelBarbosatec/tutorial_coach_mark#141]
When using an inner navigator or on more complex use cases, there is offset between the tutorial target and the real widget position if using rootOverlay.

This is a basic example reproducing the bug : 

```dart
import 'package:flutter/material.dart';
import 'package:tutorial_coach_mark/tutorial_coach_mark.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return const MaterialApp(title: 'Flutter Demo', home: MyHomePage());
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({super.key});

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  final key = GlobalKey();

  void search() {
    final target = TargetFocus(keyTarget: key, contents: const []);
    final tutorial = TutorialCoachMark(targets: [target]);
    // Using rootOverlay when showing target
    tutorial.show(context: context, rootOverlay: true);
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: const Text('tutorial_coach_mark')),
      // Added inner navigator to the app
      body: Navigator(
        pages: [MaterialPage(child: Center(child: Text('find me', key: key)))],
        onPopPage: (_, __) => true,
      ),
      floatingActionButton: FloatingActionButton(
        onPressed: () => search(),
        child: const Icon(Icons.search),
      ),
    );
  }
}
```

**Comment by guildem** [comment]
PR here : https://github.com/RafaelBarbosatec/tutorial_coach_mark/pull/142

**Comment by guildem** [comment]
Fixed in version 1.2.5
